python/lsst/ts/genericcamera/driver/alliedvisioncamera.py

```diff
@@ -232,6 +238,92 @@ def stop_live_view(self):
                 self.camera.set_pixel_format(self.normal_image_type)
         super().stop_live_view()
 
+    def _run_streaming(self) -> None:
+        """Keep streaming mode for the camera alive."""
+        with vimba.Vimba.get_instance() as v:
+            # Need to get camera again because this follows
+            # start_streaming_mode and the context manager drops out
+            # rendering self.camera inoperable
+            camera = v.get_camera_by_id(self.id)
+            # Seem to need a log statement here for things to work
+            # Otherwise the context manager fails
+            self.log.info("Running _run_streaming")
+            with camera:
```

Comment:
Does combining lines 247 and 251 as follows help?
```
with v.get_camera_by_id(self.id) as camera:
```
or is that log statement still needed?

Reply:
Also, why not set the instance to `self.camera` but create a new local variable instead?

Reply:
I've cleaned this up a bit, but the logging statement is necessary. If I remove it everything stops working. In the INFO mode of the camera logging, the camera tells me it can't access the camera with the given mode, but mode it's saying is invalid is one of that valid modes, so the error makes no sense. If I bump that up to TRACE, everything starts working. I'm chalking it up to a weird bug/interaction with the C API from the python layer.